src/store/modules/layers.store.js

```diff
@@ -163,27 +171,26 @@ const actions = {
         const activeLayerBeforeConfigChange = Array.from(state.activeLayers)
         commit('clearLayers')
         commit('setLayerConfig', config)
-        activeLayerBeforeConfigChange.forEach((layer) => {
+        for (const layer of activeLayerBeforeConfigChange) {
```

Comment:
I'm not too big of a fan to use `for` loops when we have the `.forEach` function with arrays, but I don't mind.
It doesn't help the readability IMHO, did you find some improvement in performance with this notation?

Reply:
@pakb It's a leftover of a previous implementation where I updated the metadata hier and needed to break the loop in case of error, but I reverted that part, I can also revert to use the forEach. The only advantage of for over forEach is that you can break the loop in the first one if needed (although you can use `every()` for that purpose).

Reply:
Done